.travis.yml

```diff
@@ -2,7 +2,23 @@ language: java
 # OpenTripPlanner requires Java 8 and Travis doesn't (yet) support OpenJDK 8
 jdk:
   - oraclejdk8
-  
+  - openjdk8
+  - oraclejdk9
+  - openjdk9
+# deprecated - oraclejdk10
```

Comment:
Why is this line still present? What does it mean that the Oracle JDK 10 is deprecated?

Reply:
Also, please add a comment explaining that listing many JDKs here will cause a separate build on each one of these JDKs.

Reply:
Perhaps the easiest is to point to travis documentation here
The point of putting oraclejdk10 in comment here, is to avoid having someone coming along and add it while thinking it is missing.
And indeed,  oraclejdk is now deprecated by Oracle, it is not anymore available for download